honeycombio/intro-to-o11y-nodejs
# Augmenting the Gitpod instructions a bit

## Which problem is this PR solving?

Nothing too major, just elaborating a bit on the Gitpod instructions and adding a screenshot with how to access the server in Gitpod.

## Short description of the changes

## How to verify that this has the expected result

Run through the instructions for Gitpod setup and verify traces make it into Honeycomb.
